spec/features/trade_logs/manage_trade_log_entry_spec.rb

```diff
@@ -1,92 +1,89 @@
-require 'rails_helper'
+require "rails_helper"
 
-RSpec.feature 'Trade Log Entry page', type: :feature, js: true do
+RSpec.feature "Trade Log Entry page", type: :feature, js: true do
   before(:each) do
-    @user = FactoryGirl.create(:user)
+    @user = FactoryBot.create(:user)
     login_as(@user, scope: :user)
-    @character = FactoryGirl.create(:character, user: @user)
+    @character = FactoryBot.create(:character, user: @user)
 
-    @log_entry = FactoryGirl.create(:character_log_entry, characters: [@character])
-    @magic_item = FactoryGirl.create(:magic_item, name: 'Staff of Power', rarity: 'rare', log_entry: @log_entry, character: @character)
+    @log_entry = FactoryBot.create(:character_log_entry, characters: [@character])
+    @magic_item = FactoryBot.create(:magic_item, name: "Staff of Power", rarity: "rare", log_entry: @log_entry, character: @character)
   end
 
-  scenario 'Adding trade log hides old magic item and shows new magic item' do
+  scenario "Adding trade log hides old magic item and shows new magic item", skip: true do
```

Comment:
I'd like you to take a look at these skipped scenarios, @Ariel-Thomas. My perception is that they are no longer relevant given the state of the Season 8 UI, but instead of deleting I wanted to skip them so we could revisit.

Reply:
Seems good, I'll give them a look when I have some time.